tests/e2e/cucumber/features/integrations/link.oc10.feature

```diff
@@ -0,0 +1,35 @@
+Feature: link
+
+  @issue-6239
```

Comment:
The linked issue has been resolved so after this PR has been merged we should be able to run the directory download tests for both backends (= remove the `oc10` from the filename)